client/src/utils/prettifyStatus.js

```diff
@@ -3,16 +3,40 @@ import InfoIcon from '@material-ui/icons/Info';
 import { ComponentTooltip } from '../components/generic/ComponentTooltip';
 import { Button } from '../components/generic';
 
-export const prettifyStatus = (status, id, tabValue, handleEngage, handleAcknowledge) => {
-  let firstStatus = status[0];
+/**
+ * Returns participant stats message based on the first status provided
+ *
+ * @param {boolean} isMoH - is this view displayed for MoH user
+ * @param {string} status - first status defined in row.status
+ * @returns {string} specific status message based on value / capitalized status
+ */
+const getParticipantStatus = (isMoH, status) => {
+  if (status === 'rejected') return 'Archived';
+  if (status === 'ros') return 'Return of Service';
+
+  if (isMoH) {
+    if (status === 'prospecting') return 'In Progress';
+    if (status === 'interviewing') return 'In Progress (2)';
```

Comment:
In Progress (#num) signifies the number of such statuses. e.g. The participant is prospected by two or many users.

Reply:
Thanks! I'll update that